src/components/Admin/AdminLocalisations.tsx

```diff
@@ -16,7 +16,11 @@ export const AdminLocalisations: React.FC = () => {
   const [exportDisabled, setExportDisabled] = useState(true)
   const [importDisabled, setImportDisabled] = useState(true)
   const showToast = useToast({ position: topLeft })
-  const [showModalWarning, setShowModalWarning] = useState<ModalProps>({ open: false })
+  const { ConfirmModal: WarningModal, showModal: showWarningModal } = useConfirmationModal({
```

Comment:
This is very cool, returning component from the hook, although I don't know about react internals enough to figure out if there are any side effects. I was suggesting context, but I guess this should work pretty well, if we have any glitches for any reason would suggest changing to context